caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -795,9 +906,9 @@ def fit_online(self, **kwargs):
         self.estimates.bl = [osi.b for osi in self.estimates.OASISinstances] if hasattr(
             self, 'OASISinstances') else [0] * self.estimates.C.shape[0]
         if self.params.get('online', 'save_online_movie'):
-            out.release()
+            out.release() 
```

Comment:
why are we adding these extra spaces at the end of the line here and two lines below?